src/gatsby-browser.js

```diff
@@ -28,12 +28,9 @@ const injectPinterestScript = ({ saveButton = false }) => {
 let injectedPinterestScript = false;
 
 exports.onRouteUpdate = (args, pluginOptions = {}) => {
-  const hover = Boolean(pluginOptions.saveButton)
+  const hover = Boolean(pluginOptions.saveButton);
 
-  const querySelectors = [
-    "[data-pin-do]",
-    hover ? "img" : "",
-  ]
+  const querySelectors = ["[data-pin-do]", hover ? "img" : ""]
```

Comment:
This doesn't look consistent with the other changes, there's no semicolon here although there were added elsewhere. Is this expected?

Reply:
We do have a `.filter(Boolean)` on the line below, so this is expected indeed

Reply:
🤦